subsystem/29.0.0/src/main/java/org/wildfly/legacy/test/controller/subsystem_29_0_0/TestModelControllerService29_0_0.java

```diff
@@ -112,7 +112,7 @@ protected void initCoreModel(ManagementModel managementModel, Resource modelCont
 
     @Override
     protected void preBoot(List<ModelNode> bootOperations, boolean rollbackOnRuntimeFailure) {
-        mainExtension.initialize(extensionRegistry.getExtensionContext("Test", getRootRegistration(), ExtensionRegistryType.SERVER));
+        mainExtension.initialize(extensionRegistry.getExtensionContext("Test", getRootRegistration(), ExtensionRegistryType.MASTER));
```

Comment:
@kabir This change fixed the problem I was getting running Elytron transformers against an EAP 8 version. As I understand it, for EAP 8 running in standalone mode, it is required to have a resolver extension registry registered:

https://github.com/jbossas/wildfly-core-eap/blob/9211948c3ccca80e2754e66a035e23fb8a3137f2/controller/src/main/java/org/jboss/as/controller/extension/ExtensionRegistry.java#L678-L682

However, that is not required when using a Domain Mode extension registry. I don't know if this is the correct fix or we should get the process type and decide based on this, in which case we should modify the AdditionalInitialization on the transformers tests together with a change in this class that chooses the appropriated ExtensionRegistryType based on the process type.

So far I have only found Elytron transformers requiring this sort of configuration. 

I don't know if it could be a good idea to release legacy tests 8.0.2 as it is with this PR so we can run a complete CI of WildFly and WildFly Core with this version to ensure no other tests are broken. 

Do you have any suggestion?